ReactJS-Project-CardinalKit/src/components/ManageUsers.jsx

```diff
@@ -0,0 +1,103 @@
+        <div class="px-4 py-3 mb-8 bg-white rounded-lg shadow-md dark:bg-gray-800">
+          <label class="block text-sm">
+            <span class="text-gray-700 dark:text-gray-400">First Name</span>
+            <input
+              class="block w-full mt-1 text-sm dark:border-gray-600 dark:bg-gray-700 focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:text-gray-300 dark:focus:shadow-outline-gray form-input"
+              placeholder="Jane Doe"
+              name="firstName"
+              value={this.state.firstName}
+              onChange={e => this.handleChange(e.target.name, e.target.value)}
+            />
+          </label>
+          <br />
+          <br />
+          <label class="block text-sm">
+            <span class="text-gray-700 dark:text-gray-400">Last Name</span>
+            <input
+              class="block w-full mt-1 text-sm dark:border-gray-600 dark:bg-gray-700 focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:text-gray-300 dark:focus:shadow-outline-gray form-input"
+              placeholder="Jane Doe"
+              name="lastName"
+              value={this.state.lastName}
+              onChange={e => this.handleChange(e.target.name, e.target.value)}
+            />
+          </label>
+          <br />
+          <br />
+          <label class="block text-sm">
+            <span class="text-gray-700 dark:text-gray-400">Email</span>
+            <input
+              class="block w-full mt-1 text-sm dark:border-gray-600 dark:bg-gray-700 focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:text-gray-300 dark:focus:shadow-outline-gray form-input"
+              placeholder="JaneDoe@gmail.com"
+              name="email"
+              value={this.state.email}
+              onChange={e => this.handleChange(e.target.name, e.target.value)}
+            />
+          </label>
+          <br />
+          <br />
+          <label class="block text-sm">
+            <span class="text-gray-700 dark:text-gray-400">Password</span>
+            <input
+              class="block w-full mt-1 text-sm dark:border-gray-600 dark:bg-gray-700 focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:text-gray-300 dark:focus:shadow-outline-gray form-input"
+              placeholder=""
+              name="password"
+              type='password'
+              value={this.state.password}
+              onChange={e => this.handleChange(e.target.name, e.target.value)}
+            />
+          </label>
+          <br />
+          <br />
+          <br />
+          <button
+            onClick={() => this.onSubmit()}
+            class="flex items-center justify-between px-4 py-2 text-sm font-medium leading-5 text-white transition-colors duration-150 bg-purple-600 border border-transparent rounded-lg active:bg-purple-600 hover:bg-purple-700 focus:outline-none focus:shadow-outline-purple"
```

Comment:
should always use `className` instead of `class` for a React component. and try avoiding multiple classes that exceeds 3-4 class names.

Reply:
@vjbhaskar-ui We changed `class` to  `className`, We will check how we can avoid such multiple classes.